sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2166,7 +2166,7 @@ class DataSourceV2SQLSuite
     val e = intercept[AnalysisException] {
       sql("DESCRIBE FUNCTION testcat.ns1.ns2.fun")
     }
-    assert(e.message.contains("DESCRIBE FUNCTION is only supported in v1 catalog"))
+    assert(e.message.contains("Function command is only supported in v1 catalog"))
```

Comment:
Modify the error msg because of sql param is dropped.

Reply:
Why did you drop the param? The previous message looks better.

Reply:
It looks like a regression to me, too.
Could you try not to change these parts, @ulysses-you ?

Reply:
comment at https://github.com/apache/spark/pull/28840#discussion_r442706595.

Seems can't keep the part, is there any idea ?